.github/workflows/ci.yml

```diff
@@ -80,7 +83,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: EmbarkStudios/cargo-deny-action@v2
+      - name: Run cargo-deny
+        uses: EmbarkStudios/cargo-deny-action@v2
+        with:
+          command: check
```

Comment:
By default it runs `--all-features` and it was giving an error for using the library `sp-weights` with multiple times:
```
sp-weights = { version = "31.0.0", default-features = false }
sp-weights_inkv6 = { version = "31.1.0", package = "sp-weights", default-features = false }
```